rollup.config.mjs

```diff
@@ -25,6 +29,10 @@ export default {
         }
     ],
     plugins: [
+        replace({
+            '__VERSION__': version,
+            preventAssignment: true
```

Comment:
this is required by the warning from the package. more info here: https://www.npmjs.com/package/@rollup/plugin-replace